src/components/input-dropdown/index.jsx

```diff
@@ -7,19 +7,26 @@ import Dropdown from '../dropdown'
 
 import './_base.scss'
 
+const noop = () => {}
 
 class InputDropdown extends React.PureComponent {
     constructor(props) {
         super(props)
 
-        this.onChange = this.onChange.bind(this)
-        
+        this.blur = this.blur.bind(this)
+        this.change = this.change.bind(this)
+        this.click = this.click.bind(this)
+
         this.state = {
             value: ''
         }
     }
 
-    onChange(selectedItem) {
+    blur(e) {
+        this._dropdownComponent.click()
```

Comment:
because the focus is on the input clicking outside of the dropdown list wouldn't do anything, the list would remain in place